httplog/httplog.go

```diff
@@ -37,6 +39,66 @@ type Logger struct {
 	mode Mode
 }
 
+type Config struct {
+	ModeOptions []map[string]Mode
+	defaults    map[string]Mode
+}
```

Comment:
I don't like the structure of it. 

Notice that in all other places we don't have `defaults` field. We simply set the values, thus I'd propose something like that

```go
type ModuleMode struct {
  module string
  mode Mode
}

type Config []ModuleMode
```
Default value is the one you set at the beginning of the command.

Reply:
I'm also unsure whether this `Config` should live here. It only collects `http log.Mode`. Maybe `config.go` (forwarder root dir) would be better place.

Reply:
Since it needs to parse the log modes it should be where the modes are defined IMO.

Reply:
Ok, thanks for the suggestion. I implemented it as you described but realized we also need to check for valid module names when parsing, so I also added a slice to track valid module names. WDYT?

```go
type Config struct {
       ModuleModes []ModuleMode
       modules     []string
}
```